backend/dataall/cdkproxy/stacks/dataset.py

```diff
@@ -98,8 +98,7 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
 
         quicksight_default_group_arn = None
         if env.dashboardsEnabled:
-            quicksight_default_group = Quicksight.create_quicksight_group(AwsAccountId=env.AwsAccountId)
-            quicksight_default_group_arn = quicksight_default_group['Group']['Arn']
+            quicksight_default_group_arn = f"arn:aws:quicksight:{dataset.region}:{dataset.AwsAccountId}:group/default/{Quicksight._DEFAULT_GROUP_NAME}"
```

Comment:
We cannot assume that the QS default group is already created, if dashboards was just enabled in the environment the QS group won't exist

Reply:
Yes exactly - as part of Dataset's `create_dataset()` resolver we check the QuickSight group and create the group if it does not exist already. It would be redundant to have it both there and in the cdkproxy Dataset stack

The reason it is included in this PR is because the cdkproxy would ONLY need internet access via NAT to communicate with QuickSight. If we remove here - the security group for cdkproxy can speak to the remaining services via PrivateLink and VPC Endpoints with no traffic via the internet